lib/shortcuts/lambda.js

```diff
@@ -93,8 +93,8 @@ class Lambda {
       },
       AlarmActions = [],
       Period = 60,
-      EvaluationPeriods = 1,
```

Comment:
I think you can you put an expression here that relies on the other variables which are being assigned values?

```
EvaluationPeriods = Math.ceil(Timeout / Period)
```

If that works you don't need any of the code from L106-114

Reply:
nice, do you want to keep the error if the supplied configuration values result in a alarm evaluation window shorter than the lambda timeout?

Reply:
Ok sounds good.